### PR TITLE
Update saddle points

### PR DESCRIPTION
**Pull request by rpottsoh**
@kotp what do you think?  The up shot of this commit is how to deal with a construct (TTuple in this case) that I have declared in the test runner.  Is what I have done acceptable?  Would it be better to move TTuple to its own unit that both the test runner and solution units use?  Should I provide a hint/tip at the end of the test runner that explains to the student that they need to utilize TTuple and how to make it available to the code they write?

### CODE REVIEW

**Comment by rpottsoh** [comment]
Certainly.  It is at the bottom like it is in hello-world for Ruby.  When I ported from Ruby I just mimicked it.  At the time I just figured that was the expected way to do things.  I just carried on for saddle-points. 


Sent from my android device.

**Comment by kotp** [comment]
I need to consider this in the Ruby track as well... What do you think... is it in the right place?  Is it presented a little late?

**Comment by kotp** [comment]
Hmm... I think maybe the Hello World was just small enough it did not present the same perception.

**Comment by rpottsoh** [comment]
Thinking back to august when I did the hello-world exercise obviously it failed to compile the first time.  I did not read up on the exercise on the website.  It wasn't until I decided to open up the runner that I eventually happened upon the help at the bottom.   I am trying to recall but is there some brief comments about settings that can be made to add color to the runner output at the top?  I've not read your next comment yet but at this moment I guess I would be more in favor of instructions at the top.


Sent from my android device.

**Comment by rpottsoh** [comment]
I can understand that especially if you don't have to scroll far to see them.  Or if your screen is large enough everything appears at one time.


Sent from my android device.

**Comment by kotp** [comment]
In the Ruby track, I do mention running the tests with color.  Is this what you are asking about?  It [is here](http://exercism.io/languages/ruby/tests) and has suggestions about how the tests can be enabled (the skips disabled) and how you can customize the running of the tests.

**Comment by rpottsoh** [comment]
@kotp I am getting the feeling that there is too much instruction at the top of the file now.  Some of the instruction is generic and will apply to all exercises.  Should it be included at the top on all exercises or should it be included in one of the instruction docs for this track?

File: exercises/saddle-points/uSaddlePointsExample.pas [hunk 1]
```diff
@@ -1,3 +1,14 @@
...
+// For this example solution I utilized Stefen Glienke's
+// open-source code library titled Spring4D.  Latest
+// version is available here: https://bitbucket.org/sglienke/spring4d
+// Spring4D is not absolutely necessary, but it does greatly help
+// facilitate a much simpler solution.
+//
+// Spring4D does include a Tuple record type that I have purposely
+// not used in favor of my own, simple, TTuple that I declared in the
+// test runner, in order to remove Spring4D dependency for test execution 
+//========================================================================
```
Comment (kotp) [inline exercises/saddle-points/uSaddlePointsExample.pas]: These 11 lines are fine...

File: exercises/saddle-points/uSaddlePointsTests.pas [hunk 2]
```diff
@@ -1,3 +1,35 @@
...
+// The beginning of the new unit file should contain a unit statement:
+// unit uSaddlePoints;
+//
+// The new unit should contain Interface, Implementation, and End. statements.
+//
+// Hint: Delphi will take care of all this if you instruct it to add a new unit
+// to your project.  Be sure to save the new unit as uSaddlePoints.pas before
+// trying to compile again.
+//
+// Hint: You will be using TTuple in your solution.  TTuple is
```
Comment (kotp) [inline exercises/saddle-points/uSaddlePointsTests.pas]: From lines 1 to line 24, I think that unless this is the 2nd or maybe 3rd time, the hand holding can be here, but at some point, this information should be inferred by the compile and test failure messages.

The information about the TTuple is the "new information" and so it is good here.

**Comment by kotp** [comment]
Also can squash the commits... :)

**Comment by rpottsoh** [comment]
Of course I'll squash'em!  :smiley:   Thanks

**Pull request merged**